src/Plugins/RFC.php

```diff
@@ -207,6 +191,87 @@ public function getRFC(Command $command)
         return $this->chatClient->postMessage($command->getRoom(), $message);
     }
 
+    public function getRFCVotes(Command $command)
+    {
+        $rfc = $command->getParameter(0);
+
+        if ($rfc === null) {
+            return $this->chatClient->postMessage($command->getRoom(), "Usage: `!!voting <id>`");
+        }
+
+        $uri = self::BASE_URI . '/' . urlencode($rfc);
+
+        /** @var HttpResponse $response */
+        $response = yield $this->httpClient->request($uri);
+
+        if ($response->getStatus() !== 200) {
+            return $this->chatClient->postMessage($command->getRoom(), "Nope, we can't have nice things.");
+        }
+
+        $messages = $this->prepareVotes($response->getBody(), $uri);
+
+        if(count($messages) === 0) {
+            return $this->chatClient->postMessage($command->getRoom(), "No votes found.");
+        }
+
+        return $this->chatClient->postMessage($command->getRoom(), implode("\n", array_map(function($message) {
+
+            $voters = implode("\n", array_map(function($voter) {
+
+                if($voter['voted'] === false) {
+                    return sprintf('%s has not voted', $voter['name']);
+                }
+
+                return sprintf(
+                    '%s ⭢ %s',
```

Comment:
Firstly I'd generally rather avoid using multi-byte literals in source code, in order to avoid any weirdness that may arise from someone's editor automagically and transparently changing the encoding.

Secondly, pursuant to the first point, we have https://github.com/Room-11/Jeeves/blob/master/src/Chat/Client/Chars.php for this exact reason, and it already has a suitable character (`RIGHTWARDS_ARROW`) which is known to work sanely in all browsers we care about.